cdap-common/bin/common.sh

```diff
@@ -272,6 +273,10 @@ cdap_set_hive_classpath() {
         # tez-site.xml also need to be passed to explore service
         EXPLORE_CONF_FILES=${EXPLORE_CONF_FILES}:${TEZ_CONF_DIR}/tez-site.xml:
       fi
+      if [[ "${HIVE_EXEC_ENGINE}" == "spark" ]]; then
```

Comment:
This only enforces spark when the execution engine is spark, since it would mean a broken default.